report/src/test/java/com/hartwig/actin/report/pdf/TestReportWriterApplication.java

```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) throws IOException {
 
     @NotNull
     private static Report createTestReport() throws IOException {
-        Report report = TestReportFactory.createProperTestReport();
+        Report report = TestReportFactory.createExhaustiveTestReport();
```

Comment:
Maybe we should have different functions for "proper test reports" and "exhaustive test reports". The report that can be downloaded from our github page is the one with "proper" data, which does not necessarily have an event of every type

Reply:
I'll change this back.